src/storage/src/hummock/compactor/compaction_filter.rs

```diff
@@ -120,3 +120,19 @@ impl MultiCompactionFilter {
         self.filter_vec.push(filter);
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::HashMap;
+
+    use risingwave_common::catalog::TableId;
+    use risingwave_hummock_sdk::key::{FullKey, TableKey};
+
+    use super::{CompactionFilter, TtlCompactionFilter};
+
+    #[test]
+    fn test_ttl_u32() {
+        let mut ttl_filter = TtlCompactionFilter::new(HashMap::from_iter([(1, 4000000000)]), 1);
+        ttl_filter.should_delete(FullKey::new(TableId::new(1), TableKey(vec![]), 1).to_ref());
```

Comment:
shall we need to assert the result of `should_delete`?

Reply:
> shall we need to assert the result of `should_delete`?

In debug mode, rust will panic when overflow.